### ISSUE TITLE
add action that installs and tests pyopenms nightly conda package on different platforms

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/OpenMS#6712]
@jpfeuffer
would this entail just using conda to install pyopenms and then executing the test script?

**Comment by jpfeuffer** [comment]
Yes. Same for pypi maybe.

**Comment by jpfeuffer** [comment]
By the way, there is of course no test script after install anymore.

**Comment by jpfeuffer** [comment]
Ah. Saw your other issue.

**Comment by timosachsenberg** [comment]
see more details in @matteopilz issue

**Comment by enetz** [comment]
@timosachsenberg  where exactly are our nightly conda packages?
Is this it? https://anaconda.org/OpenMS/pyopenms

**Comment by timosachsenberg** [comment]
btw Matteo already has a nice action that you probably can reause

**Comment by enetz** [comment]
yes, looks like I will only have to exchange the package source and make sure miniconda is installed instead of pip

**Comment by timosachsenberg** [comment]
@jpfeuffer how to move forward here? dump the PR and wait for bioconda utils to incorporate your patch?

**Comment by timosachsenberg** [comment]
Maybe I got it wrong but I thought we want to only rely on the bioconda test and thus the PR is not needed?

**Comment by jpfeuffer** [comment]
I said we could. If you agree we can close the PR.

**Comment by timosachsenberg** [comment]
then let's keep it as simple as possible and close the PR (sorry Eugen)
